src/At.vue

```diff
@@ -46,10 +46,6 @@ export default {
       type: Boolean,
       default: true
     },
-    members: {
-      type: Array,
-      default: () => []
-    },
```

Comment:
@piscitelli91 I'm confused why the prop of `members` is deleted 😕 it will break the members passing in

Reply:
You are right, i knought that the pull request send my branch at the date of pull request and not for future commits (I'm working on new functions that I will discuss with you).

I will create a new branch until this commit: 9196ffe7d0d658236772f76e7bbe6f67cb59e006
Sorry for mistake :)

Reply:
@piscitelli91 that would be cool, otherwise, I'm also trying to merge with fix if possible